_episodes_rmd/05-r-markdown.Rmd

```diff
@@ -35,44 +31,109 @@ set.seed(20210109)
-and that it had 2 outliers? Which countries are those?
+Remember that yesterday we made a scatter plot of year vs. population, separated into a plot for each contient, and that it had 2 outliers:
+
+```{r, message = FALSE}
+library(tidyverse)
+smoking <- read_csv('data/smoking_cancer.csv')
+
+smoking %>% 
+  ggplot(aes(x=year,y=pop)) +
+  geom_point() +
+  facet_wrap(vars(continent))
+```
+
+Write some code to figure out which countries these are (even if you already know!).
+
+> ## Solution
+> ```{r}
+> smoking %>% filter(pop > 5e8) %>% select(country) %>% unique()
+> ```
+> Here we introduced a new function: `unique()`. 
+> This function is not required to find the answer to this question, but it helps us get the answer a bit more quickly.
+{: .solution}
+
+Next, plot year vs. population separated into a plot for each continent but excluding the 2 outlier countries. Note that usually you don't want to exclude certain data points from a plot because it is misleading (see Bonus 2 for an alternative). 
+
+> ## Solution
+> ```{r}
+> smoking %>% 
+> filter(country != 'China') %>% 
+> filter(country != 'India') %>% 
+> ggplot(aes(x=year,y=pop)) +
+> geom_point() +
+> facet_wrap(vars(continent))
+> ```
+> Another solution is to use only one filter command and separate the two true/false statements with an ampersand (`&`), which means that you want to exclude both China and India:
+> ```{r}
+> smoking %>% 
+> filter(country != 'China' & country != 'India') %>% 
+> ggplot(aes(x=year,y=pop)) +
+> geom_point() +
+> facet_wrap(vars(continent))
+> ```
+{: .solution}
+
+Bonus 1: Instead of hard-coding the two countries to remove them, remove the two outliers by combining your solutions to the first two questions.
 
 > ## Solution
 > ```{r}
-> gapminder %>% filter(pop > 1e9) %>% select(country) %>% unique()
+> smoking %>% 
+> filter(pop < 5e8) %>% 
+> ggplot(aes(x=year,y=pop)) +
+> geom_point() +
+> facet_wrap(vars(continent))
 > ```
 {: .solution}
 
-Next, plot year vs. population separated into a plot for each continent but excluding the 2 outlier countries.
+Bonus 2: How can you make the differences between countries more visible on the plot without excluding the two countries you identified above?
 
 > ## Solution
+> You can scale the y axis using a log10 scale to make the differences more visible:
 > ```{r}
-> gapminder %>% filter(country != 'China' & country != 'India') %>% ggplot(aes(x=year,y=pop)) +
+> smoking %>% 
+> ggplot(aes(x=year,y=pop)) +
 > geom_point() +
+> scale_y_log10() +
 > facet_wrap(vars(continent))
 > ```
 {: .solution}
 
-## What is R Markdown and why use it? {##why-use-r-markdown}
+
+## What is R Markdown and why use it?
 _[Back to top](#contents)_
 
-Recall that our  goal is to generate a report to the United Nations on how a country's life expectancy is related to GDP.
+Recall that our goal is to generate a report to the United Nations on how a country's smoking rate is related to its lung cancer rate.
```

Comment:
I mean...we can say United Nations, but I don't think it's necessary? It's fun in a way I guess.
```suggestion
Recall that our goal is to generate a report on how a country's smoking rate is related to its lung cancer rate.
```